Basic-Car-Maintenance/Shared/Managers/AnayticsManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AnayticsManager.swift
```

Comment:
Rename this file, it's spelled wrong, and be sure this header comment is updated too